### PR TITLE
Improve the startup logging and fix the broken git version logging

### PR DESCRIPTION
**Pull request by mayhem**
as it says on the tin..

### CODE REVIEW

File: metabrainz/__init__.py [hunk 1]
```diff
@@ -56,23 +55,23 @@ def create_app(config_path=None):
 
         app.config.from_pyfile(consul_config, silent=True)
 
+    # Printing out some debug values such as config and git commit
+    try:
+        with open(".git-version") as f:
+            git_version = f.read()
+        print('Running on git commit %s' % git_version.strip())
+    except subprocess.CalledProcessError as e:
```
Comment (paramsingh) [inline metabrainz/__init__.py]: This error will probably not be thrown now.

File: metabrainz/__init__.py [hunk 2]
```diff
@@ -56,23 +55,23 @@ def create_app(config_path=None):
 
         app.config.from_pyfile(consul_config, silent=True)
 
+    # Printing out some debug values such as config and git commit
+    try:
+        with open(".git-version") as f:
```
Comment (paramsingh) [inline metabrainz/__init__.py]: We could do this in `utils.get_git_commit`.

File: metabrainz/__init__.py (hunk 1 above)
Comment (mayhem) [reply metabrainz/__init__.py]: Oh, good point.

File: metabrainz/__init__.py (hunk 2 above)
Comment (mayhem) [reply metabrainz/__init__.py]: Remind me, was this added for just this case or was this an existing function that works in other contexts?

**Comment by mayhem** [comment]
An env variable would make it available to the Dockerfile, but not the running container in production. The Dockerfile would need to write it to a file, so its not much of a difference.

File: metabrainz/__init__.py (hunk 2 above)
Comment (paramsingh) [reply metabrainz/__init__.py]: No, just added here, so could be removed easily.

Comment (mayhem) [reply metabrainz/__init__.py]: Done!

**Pull request merged**